editor/src/third-party/remix/flat-routes.ts

```diff
@@ -0,0 +1,542 @@
+/* eslint-disable */
+
+// Copy pasted and adapted from Remix: https://github.com/remix-run/remix/blob/main/packages/remix-dev/config/flat-routes.ts
```

Comment:
Can you update these URLs to use the permalink (i.e. including the commit SHA) URLs please